inf-clojure.el

```diff
@@ -1254,15 +1254,43 @@ See variable `inf-clojure-buffer'."
   "Return DATA if and only if it is a list."
   (when (listp data) data))
 
-(defun inf-clojure-completions (expr)
-  "Return a list of completions for the Clojure expression starting with EXPR."
+(defun inf-clojure-list-completions (expr)
+  "Return completions for the Clojure expression starting with EXPR.
+
+Its only ability is to parse a list of candidate strings, every
+other EXPR will be discarded and nil will be returned."
   (when (not (string-blank-p expr))
     (thread-first
         (format (inf-clojure-completion-form) (substring-no-properties expr))
       (inf-clojure--process-response (inf-clojure-proc) "(" ")")
       (inf-clojure--read-or-nil)
       (inf-clojure--list-or-nil))))
 
+(defcustom inf-clojure-completions 'inf-clojure-list-completions
```

Comment:
Won't `inf-clojure-completions-fn` (or something like this) be a better name?

Reply:
I like this better yes